src/NexusMods.App.UI/TreeNodeVM.cs

```diff
@@ -57,6 +67,14 @@ public TreeNodeVM(Node<TItem, TKey> node, TreeNodeVM<TItem, TKey>? parent = null
         Item = node.Item;
         Id = node.Key;
         Parent = parent;
+        
+        this.WhenActivated(d =>
+        {
+            // re-broadcast 'PropertyChanged' event from the inner item.
+            Item.WhenAnyValue(x => x.IsExpanded)
+                .Subscribe(_ => this.RaisePropertyChanged(nameof(IsExpanded)))
+                .DisposeWith(d);
+        });
```

Comment:
@erri120 Wanted to check in with you to see if there was a better suited way to handle this.
Basically we need to have `IsExpanded` duplicated both on the flat item and the tree wrapper (because treeDataGrid source can't bind to the internal item...)

Is there a clean way with reactive ui to do a two way bind between two view models? Or is the approach here ok?